added check for secondaryMonitor before accessing members

seanbourg opened the pull request:
Fixed issue encountered in Gnome 42 where the secondary monitor was causing an issue on user login. Checking for secondary monitor prior to accessing members fixes the issue.

seanbourg committed:
added check for secondaryMonitor before accessing members

seanbourg commented:
![Screenshot from 2022-03-16 11-38-51](https://user-images.githubusercontent.com/2058138/158645006-3653395e-ac96-4dfd-8943-de9b728dc002.png)

axxapy commented on src/modScaleThumbnails.js:
```diff
@@ -9,18 +9,20 @@ var Mod = class extends mod.Base {
-        SecondaryMonitorDisplay.prototype._getThumbnailsHeight = function(box) {
-            if (!this._thumbnails.visible)
-                return 0;
+        if (SecondaryMonitorDisplay) {
```
Could you please return instead of making nested code?
```suggestion
        if (!SecondaryMonitorDisplay) return; // gnome 42: this object does not exist when there is only one monitor
```

Thanks

seanbourg committed:
Update modScaleThumbnails.js

Modified to return if SecondaryMonitorDisplay when it is not found. Removed second check (covered by function call).

axxapy commented on src/modScaleThumbnails.js:
```diff
@@ -8,19 +8,19 @@ var Mod = class extends mod.Base {
-
+        
         // Thumbnails on second monitor
+        if (!SecondaryMonitorDisplay) return;
```
Looks great!

May I also ask you to remove all other changes except for this line? (as no other changes actually change anything).

Thank you

seanbourg committed:
Removed local version changes.

seanbourg commented:
Updated: removed my local revisions.

axxapy commented:
Thanks.
Changes are on review, will be published soon: https://extensions.gnome.org/review/30252
